<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1056</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>2</y>
      <width>731</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="halls_btn">
         <property name="text">
          <string>halls</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="films_btn">
         <property name="text">
          <string>films</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTableWidget" name="table_place">
         <column>
          <property name="text">
           <string>id</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>seats</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QToolBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>40</y>
      <width>271</width>
      <height>361</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page_27">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>271</width>
       <height>268</height>
      </rect>
     </property>
     <attribute name="label">
      <string>ADD</string>
     </attribute>
     <widget class="QToolBox" name="toolBox_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>271</width>
        <height>271</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="add_hall">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>271</width>
         <height>209</height>
        </rect>
       </property>
       <attribute name="label">
        <string>add_hall</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget_8">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>251</width>
          <height>101</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_21">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_22">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_26">
             <item>
              <widget class="QLabel" name="label_19">
               <property name="text">
                <string>name: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="hall_name"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_27">
             <item>
              <widget class="QLabel" name="label_20">
               <property name="text">
                <string>size:   </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="hall_seats">
               <property name="maximum">
                <number>1000</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="add_hall_btn">
           <property name="text">
            <string>add</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="page_33">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>271</width>
         <height>209</height>
        </rect>
       </property>
       <attribute name="label">
        <string>add_film</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget_9">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>244</width>
          <height>181</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_23">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_28">
           <item>
            <widget class="QLabel" name="label_21">
             <property name="text">
              <string>name:      </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="film_name"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_29">
           <item>
            <widget class="QLabel" name="label_22">
             <property name="text">
              <string>genre:      </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="genre"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_30">
           <item>
            <widget class="QLabel" name="label_23">
             <property name="text">
              <string>ticket_cost:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="ticket_cost">
             <property name="maximum">
              <number>1000</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_31">
           <item>
            <widget class="QLabel" name="label_24">
             <property name="text">
              <string>duration:   </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="duration">
             <property name="maximum">
              <number>1000</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_32">
           <item>
            <widget class="QLabel" name="label_25">
             <property name="text">
              <string>hall:          </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="hall">
             <property name="maximum">
              <number>1000</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="add_film_btn">
           <property name="text">
            <string>add</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_29">
     <attribute name="label">
      <string>UPDATE</string>
     </attribute>
     <widget class="QToolBox" name="update_toolbox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>271</width>
        <height>271</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="update_hall">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>271</width>
         <height>209</height>
        </rect>
       </property>
       <attribute name="label">
        <string>update_hall</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget_6">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>261</width>
          <height>101</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_18">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>hall_id:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="id_hall_update">
             <property name="maximum">
              <number>1000</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="update_hall_btn">
             <property name="text">
              <string>update</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_11">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_12">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>new name: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="new_hall_name"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_13">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>new size:   </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="new_hall_seats">
               <property name="maximum">
                <number>1000</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="update_film">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>271</width>
         <height>209</height>
        </rect>
       </property>
       <attribute name="label">
        <string>update_film</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget_7">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>241</width>
          <height>197</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_20">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>film_id:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="id_film_update">
             <property name="maximum">
              <number>1000</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="update_film_btn">
             <property name="text">
              <string>update</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_18">
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>new name:      </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="new_film_name"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_21">
           <item>
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>new genre:      </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="new_genre"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_23">
           <item>
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>new ticket_cost:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="new_ticket_cost">
             <property name="maximum">
              <number>100000</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_22">
           <item>
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>new duration:   </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="new_duration">
             <property name="maximum">
              <number>1000</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_24">
           <item>
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>new hall:          </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="new_hall">
             <property name="maximum">
              <number>1000</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_28">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>271</width>
       <height>268</height>
      </rect>
     </property>
     <attribute name="label">
      <string>DELETE</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>221</width>
        <height>91</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>hall_id:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="id_hall_delete">
           <property name="maximum">
            <number>1000</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="delete_hall_btn">
           <property name="text">
            <string>delete</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>film_id:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="id_film_delete">
           <property name="maximum">
            <number>1000</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="delete_film_btn">
           <property name="text">
            <string>delete</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
